packages/global-styles-ui/src/global-styles-ui.tsx

```diff
@@ -240,8 +240,6 @@ export function GlobalStylesUI( {
 						</GlobalStylesNavigationScreen>
 					) ) }
 
-					<ContextScreens />
```

Comment:
Should we also remove https://github.com/WordPress/gutenberg/blob/trunk/packages/global-styles-ui/src/global-styles-ui.tsx#L89-L93? -cc @youknowriad ?

Reply:
> Should we also remove https://github.com/WordPress/gutenberg/blob/trunk/packages/global-styles-ui/src/global-styles-ui.tsx#L89-L93?

Good catch, I believe the screen is not necessary. This is because the color palette cannot be edited at the block level.